library/src/main/java/com/eyeem/deviceinfo/Factory.java

```diff
@@ -233,4 +237,50 @@ private static DisplayMetrics getDisplayMetrics(Context context) {
       dm.setToDefaults();
       return dm;
    }
+
+   // https://stackoverflow.com/questions/20264268/how-do-i-get-the-height-and-width-of-the-android-navigation-bar-programmatically/29938139#29938139
+   public static Point getNavigationBarSize(Context context) {
+      Point appUsableSize = getAppUsableScreenSize(context);
+      Point realScreenSize = getRealScreenSize(context);
+
+      // navigation bar on the right
```

Comment:
it can be on right and on left, maybe consider using Rect since it has width and height methods and you can position it on the screen. optionally your own class with status bar orientation (Horizontal|Vertical|None) especially since you keep: `navigationBarHeight`